tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -11,6 +11,7 @@
 class Messages(Base):
 
     name = 'Messages'
+    manifest_url = "app://sms.gaiamobile.org/manifest.webapp"
```

Comment:
Python won't accept that syntax. It should be:
```python
manifest_url = '{}sms{}./manifest.webapp'.format(self.DEFAULT_PROTOCOL, self.DEFAULT_APP_HOSTNAME)
```
`/manifest.webapp` is also a constant we have to extract.